test/test_python_dispatch.py

```diff
@@ -50,7 +51,10 @@ def unwrap(e):
         def wrap(e):
             return LoggingTensor(e) if isinstance(e, torch.Tensor) else e
 
-        rs = tree_map(wrap, func(*tree_map(unwrap, args), **tree_map(unwrap, kwargs)))
+        # no_dispatch is only needed if you use enable_python_mode.
```

Comment:
In the case where the two keys would be merged, this won't be needed anymore right?

Reply:
We still need no_dispatch to prevent infinite recursion. The reason why we need it is:
- let's say the user called torch.empty
- this dispatches to this torch_dispatch item
- and then inside torch_dispatch we call torch.empty
- torch.empty is going to redispatch to the torch_dispatch, unless we add the no_dispatch